frontend/src/container/SearchResult.jsx

```diff
@@ -230,21 +229,18 @@ class SearchResult extends React.Component {
         <div className='tracim__content-scrollview'>
           <PageWrapper customClass='searchResult'>
             <PageTitle
-              parentClass={'searchResult'}
+              parentClass='searchResult'
               title={currentNumberSearchResults === 1
                 ? props.t('Search result')
-                : props.t('Search results')
-              }
+                : props.t('Search results')}
               icon='search'
               subtitle={this.getSubtitle()}
               breadcrumbsList={props.breadcrumbs}
             />
 
             <PageContent parentClass='searchResult'>
-              <div className='folder__content' data-cy={'search__content'}>
-                {currentNumberSearchResults > 0 &&
-                  <ContentItemHeader showSearchDetails />
-                }
+              <div className='folder__content' data-cy='search__content'>
+                {currentNumberSearchResults > 0 && <ContentItemHeader showSearchDetails />}
```

Comment:
is
```suggestion
{currentNumberSearchResults > 0 && (
  <ContentItemHeader showSearchDetails />
)}
```
fine for the new eslint rules ?